kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
* As long as the element query breakpoints are different from window-responsive's breakpoints, you won't have `everything switch "at the same time"`. 
* If we just squeeze the content, the current design wont work. We need a different design to adapt to this guideline.